main.tf

```diff
@@ -130,15 +131,16 @@ locals {
 }
 
 module "gke_app" {
-  source  = "wandb/wandb/kubernetes"
-  version = "1.4.1"
+  # source  = "wandb/wandb/kubernetes"
+  # version = "1.4.1"
+  source = "github.com/wandb/terraform-kubernetes-wandb?ref=recreate_pod_with_new_db_connection_string"
```

Comment:
We can get rid of this, the pod will get kicked in this case because the image and tag will change.